src/plugins/dashboard/public/types.ts

```diff
@@ -212,3 +230,36 @@ export interface DashboardProvider {
   //   "http://../app/myplugin#/edit/abc123"
   editUrlPathFn: (obj: SavedObjectType) => string;
 }
+
+export interface DashboardServices extends CoreStart {
+  pluginInitializerContext: PluginInitializerContext;
+  opensearchDashboardsVersion: string;
+  history: History;
+  osdUrlStateStorage: IOsdUrlStateStorage;
+  core: CoreStart;
+  data: DataPublicPluginStart;
+  navigation: NavigationStart;
+  savedObjectsClient: SavedObjectsClientContract;
+  savedDashboards: SavedObjectLoader;
+  dashboardProviders: () => { [key: string]: DashboardProvider };
+  dashboardConfig: OpenSearchDashboardsLegacyStart['dashboardConfig'];
+  dashboardCapabilities: any;
+  embeddableCapabilities: {
+    visualizeCapabilities: any;
+    mapsCapabilities: any;
+  };
```

Comment:
likely out of scope for this PR, but can we create an issue to better type these capabilities interfaces?

Reply:
Yea will create an issue to track this

Reply:
#4052